Shared/Samples/Authenticate with Integrated Windows Authentication/AuthenticateWithIntegratedWindowsAuthenticationView.swift

```diff
@@ -0,0 +1,189 @@
+                } else {
+                    urlEntryView
+                }
+            }
+        }
+        .onDisappear {
+            Task {
+                // Reset the challenge handlers and clear credentials
+                // when the view disappears so that user is prompted to enter
+                // credentials every time the sample is run, and to clean
+                // the environment for other samples.
+                await model.teardownAuthenticator()
+            }
+        }
+        .sheet(item: $model.selectedItem) { selectedItem in
+            NavigationStack {
+                MapView(map: Map(item: selectedItem.portalItem))
+                    .navigationTitle(selectedItem.portalItem.title)
+                    .navigationBarTitleDisplayMode(.inline)
+                    .toolbar {
+                        ToolbarItem(placement: .topBarTrailing) {
+                            Button("Done") {
+                                model.selectedItem = nil
+                            }
+                        }
+                    }
+            }
+            .interactiveDismissDisabled()
+            .highPriorityGesture(DragGesture())
+            .pagePresentation()
+        }
+        .animation(.default, value: model.isConnecting)
+        .authenticator(model.authenticator)
+    }
+    
+    @ViewBuilder private var urlEntryView: some View {
+        Section {
+            HStack {
+                TextField("IWA Secured Portal URL", text: $model.portalURLString)
+                    .onSubmit { Task { await model.connectToPortal() } }
+                    .autocapitalization(.none)
+                    .keyboardType(.URL)
+                Button("Connect") {
+                    Task { await model.connectToPortal() }
+                }
+                .disabled(model.portalURL == nil)
+            }
+        }
+    }
+}
+
+/// A value that represents an item selected by the user.
+private struct SelectedItem: Identifiable {
+    /// The portal item that was selected.
+    let portalItem: PortalItem
+    
+    var id: ObjectIdentifier {
+        ObjectIdentifier(portalItem)
+    }
+}
+
+extension AuthenticateWithIntegratedWindowsAuthenticationView {
+    @MainActor
+    class Model: ObservableObject {
+        /// The authenticator to handle authentication challenges.
+        let authenticator = Authenticator(promptForUntrustedHosts: true)
+        
+        /// The URL string entered by the user.
+        @Published var portalURLString = ""
+        
+        /// The fetched portal content.
+        @Published var portalContent: Result<PortalQueryResultSet<PortalItem>, Error>?
+        
+        /// A Boolean value indicating if a portal connection is in progress.
+        @Published var isConnecting = false
+        
+        /// The selected item.
+        @Published fileprivate var selectedItem: SelectedItem?
+        
+        /// The URL to the portal.
```

Comment:
Should the be `of` the portal?
```suggestion
        /// The URL of the portal.
```